clang/include/clang/CIR/Dialect/IR/CIRTypes.td

```diff
@@ -194,9 +194,22 @@ def CIR_PointerType : CIR_Type<"Pointer", "ptr",
     `CIR.ptr` is a type returned by any op generating a pointer in C++.
   }];
 
-  let parameters = (ins "mlir::Type":$pointee);
+  let parameters = (ins "mlir::Type":$pointee, DefaultValuedParameter<"unsigned", "0">:$addrSpace);
 
-  let assemblyFormat = "`<` $pointee `>`";
+  let builders = [
```

Comment:
we don't have clang-format for tablegen just yet, please make sure this fits into 80-cols

Reply:
in the future plz don't mark as resolved, it's better if the reviewer do it so we can confirm the review was applied (it happens often that people mark as resolved and things didn't really change, because mistakes happen)